ritwik12_Celestial-bodies-detection_app_views_46.py

```
@@ -0,0 +1,107 @@
+def about():
+    return render_template("about.html")
+
+
+@app.route("/result")
+def result():
+    cwd = os.path.join(app.root_path, "..", "hub", "examples", "image_retraining")
+    try:
+        celestial_object, labels = get_labels(imageBytes, cwd)
+    except NameError:
+        return render_template("error.html", detail="You are not supposed to be here.")
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)